support/docker/php-fpm/Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN apt-get update && apt-get install -y \
 
 # Composer
 RUN curl -sL https://getcomposer.org/installer | php -- --install-dir /usr/bin --filename composer
+RUN mkdir /.composer
+RUN chmod -R a+w /.composer
```

Comment:
FYI: mkdir supports -m parameter to set permissions.

Reply:
Ah good to know that! Updated the code to be a one-liner